lib/source-map/source-node.js

```diff
@@ -375,12 +372,16 @@ define(function (require, exports, module) {
         lastOriginalSource = null;
         sourceMappingActive = false;
       }
-      chunk.match(REGEX_CHARACTER).forEach(function (ch, idx, array) {
-        if (REGEX_NEWLINE.test(ch)) {
+      for (var idx = 0, length = chunk.length; idx < length; idx++) {
+        ch = chunk[idx];
+        if (ch === "\r" && chunk[idx+1] === "\n") {
+            continue;
+        }
```

Comment:
@lydell sure thing - you mean straight up remove lines `377-379`?  Seems reasonable.